### PR TITLE
Added SV variant filter.

### PR DESCRIPTION
**Pull request by dnil**
Initiated SV variant filter!

### CODE REVIEW

**Comment by moonso** [comment]
Nice! I will have a look 😄

**Comment by dnil** [comment]
Regarding editing commits, I checked "allow edits from maintainers" - looks like this might help you edit the PR anyhow?

**Comment by dnil** [comment]
Adresses #443 3/4 items. ISCN will have to be discussed more. We could perhaps try it on a variant-by-variant basis on the pipeline or loading side first.

File: scout/server/blueprints/variants/templates/variants/sv-variants.html [hunk 1]
```diff
@@ -59,8 +95,101 @@
   <div class="container-fluid">
     {{ pagination() }}
   </div>
+</div>
 {% endblock %}
 
+{% macro filters_form() %}
```
Comment (robinandeer) [inline scout/server/blueprints/variants/templates/variants/sv-variants.html]: Perhaps this macro can be shared between the templates?

File: scout/server/blueprints/variants/forms.py [hunk 2]
```diff
@@ -41,3 +41,22 @@ class FiltersForm(FlaskForm):
 
     thousand_genomes_frequency = DecimalField('1000 Genomes', places=2)
     exac_frequency = DecimalField('ExAC', places=2)
+
+class SvFiltersForm(FlaskForm):
```
Comment (robinandeer) [inline scout/server/blueprints/variants/forms.py]: Maybe this class can inherit the common fields from another class?

We will look into it after merging this PR 👍

**Pull request merged**